pvm.py

```diff
@@ -30,18 +31,40 @@ def _init_logger():
 VIDEO_PATH = "jellyfish720p.mp4"
 media = ""
 IS_FILE_SET = False
+NEXT_TIME = None
 
 def parse_commands(*args):
 	global media
 	global VIDEO_PATH
 	global IS_FILE_SET
+	global NEXT_TIME
+	if len(args) == 4:
+		seconds = int(args[3]) + 3
+		mintues = int(args[2])
+		hours = int(args[1])
+		if seconds >= 60:
+			seconds -= 60
+			mintues += 1
+		if mintues == 60:
+			mintues = 0
+			hours += 1
+		time_data = str(hours) + ":" + str(mintues) + ":" + str(seconds)
```

Comment:
please fix `mintues` > `minutes`

Reply:
I will test the new MAX file again.
For the previous version of pvm_send. For each rpi, MAX will send the timestamp first and then send the command.

Reply:
that's what I made sure it would still happen: first send timestamp and then send command.
As it was, it was sending timestamp even when it was not sending a command, for example, when the `pvm_send` abstraction received messages to change its state.